### PR TITLE
Prepare release 2.1.0

### PR DESCRIPTION
**Pull request by d4straub**
In preparation of release 2.1.0. Includes new features, bug fixes, software updates. Details in the [CHANGELOG](https://github.com/nf-core/ampliseq/blob/dev/CHANGELOG.md).

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
    - [x] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by erikrikarddaniel — APPROVED** [review]
LGTM

File: docs/usage.md [hunk 1]
```diff
@@ -4,13 +4,134 @@
...
+The input data can be passed to nf-core/ampliseq in three possible ways using the `--input` parameter, either a folder containing zipped FastQ files, a tab-seperated samplesheet, or a fasta file to be taxonomically classified.
+
+Optionally, a metadata sheet can be specified for downstream analysis.
+
+### Direct FASTQ input
+
+The easiest way is to specify directly the path to the folder that contains your input FASTQ files. For example:
+
+```console
+--input 'path/to/data'
```
Comment (jfy133) [inline docs/usage.md:18]:
```suggestion
--input 'path/to/data/'
```

File: modules/local/picrust.nf [hunk 2]
```diff
@@ -0,0 +1,64 @@
+// Import generic module functions
```
Comment (jfy133) [inline modules/local/picrust.nf:1]: Wouldn't this make sense a proper nf-core/module? Or is that planned for a later release?

**Review by jfy133 — APPROVED** [review]
Was scared at firs with 139 files, but was mostly formatting so LGTM

File: docs/usage.md (hunk 1 above)
Comment (jfy133) [reply docs/usage.md:18]: To specify that this is a folder as input? Unless this breaks the picking up of the contained FASTQs files

File: modules/local/picrust.nf (hunk 2 above)
Comment (d4straub) [reply modules/local/picrust.nf:1]: Yes, that is the best candidate for a nf-core module in this pipeline. However, to my knowledge, it is only used with bacterial 16S amplicon sequencing (well, theoretically any marker gene), therefore pretty specific to this pipeline. But yes, if Picrust2 is used by any other pipeline, I'd make this into a proper nf-core module.

File: docs/usage.md (hunk 1 above)
Comment (d4straub) [reply docs/usage.md:18]: That should not break anything, so thats fine.

**Comment by d4straub** [comment]
@jfy133 & @erikrikarddaniel Thanks for your reviews!

**Pull request merged**